lifelines/tests/test_estimation.py

```diff
@@ -227,12 +233,12 @@ def test_weibull_model_does_not_except_negative_or_zero_values(self):
 
     def test_exponential_data_produces_correct_inference_no_censorship(self):
         wf = WeibullFitter()
-        N = 10000
+        N = 40000
         T = 5 * np.random.exponential(1, size=N) ** 2
         wf.fit(T)
         assert abs(wf.rho_ - 0.5) < 0.01
         assert abs(wf.lambda_ - 0.2) < 0.01
-        assert abs(wf.median_ - 5 * np.log(2) ** 2) < 0.01
+        assert abs(wf.median_ - 5 * np.log(2) ** 2) < 0.1  # worse convergence
```

Comment:
former. I was surprised how slow the sample median converges to the true median.